chapter2/exercise1.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+
+    displayTriangle = function() {
+        var triangle = '#';
```

Comment:
tak naprawde to sie powinno nazywac "lane" a nie "triangle", bo w tym momencie jest tylko linia

Reply:
Ok :)